### ISSUE TITLE
Bring back page thumbnails on reading pane

### ISSUE THREAD

**Issue by andrewcrook** [burtonator/polar-bookshelf#1553]
I found it really useful to look at the page thumbnails to see my progress.

![image](https://user-images.githubusercontent.com/803618/99547626-a6d62600-29af-11eb-9132-849a46a20111.png)

**Comment by burtonator** [comment]
I agree it was kind of useful but a pain to build in the new model. I'll try to play with it when I get some time though.
